### PR TITLE
DEVTOOLS: Add a helper script for adding a class to an engine

### PR DESCRIPTION
**Pull request by mduggan**
When working on new parts of an engine or new engines, the boilerplate needed to add a new
class can be a bit of a chore.  I created a simple script to save some work that others might find useful.

Comments welcome.

### CODE REVIEW

File: devtools/README [hunk 1]
```diff
@@ -174,6 +174,10 @@ ______________________________
     Achievements lists.
 
 
+make_class.py
+-------------------
+    Tool that adds all the boilerplate for a new C++ class inside an engine
+
```
Comment (sev-) [inline devtools/README:180]: Please add at least one usage example. Demonstrate creation of class files in a subdirectory for a specific engine.

Comment (mduggan) [reply devtools/README:191]: Good idea, done.

**Comment by mduggan** [comment]
Seems like no objections so I'll merge this now.

File: devtools/README [hunk 2]
```diff
@@ -174,6 +174,21 @@ ______________________________
...
 
 
+make_class.py
+-------------------
+    Tool that adds all the boilerplate for a new C++ class inside an engine
+	Examples:
+
+	$ make_class.py scumm . LeChuck
+	Make new class `LeChuck` in the scumm engine root (engines/scumm/).  Creates
+	boilerplate le_chuck.cpp and le_chuck.h containing class Scumm::LeChuck and
```
Comment (sev-) [inline devtools/README:184]: This example is a bit confusing: how the script comes with `le_chuck` (with underscore) from `LeChuck`?

Comment (mduggan) [reply devtools/README:184]: It converts camelcase class names to lowercase with underscores to follow the convention, I could mention that explicitly.

Comment (mduggan) [reply devtools/README:184]: Done in 8d0aa01b163.

**Pull request merged**